### ISSUE TITLE
Checkbox value change using keyboard is not accounced by Narrator

### ISSUE THREAD

**Issue by YingXue** [OfficeDev/office-ui-fabric-react#11134]
### Environment Information

- **Package version(s)**: 7.59.0
- **Browser and OS versions**: chrome 78.0.3904.87
I saw the note below, however we are using Fabric react library in an electron app. Wondering if there is any plan the fabric team will be fixing the issue in Chrome.

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Narrator won't anounce check box value updated using keyboard (space key or enter key)

#### Actual behavior:

Open https://developer.microsoft.com/en-us/fabric#/controls/web/checkbox in Chrome. Issue described above can be reproduced.

#### Expected behavior:

Narrator should anounce check box value updated using keyboard (space key or enter key)

**Comment by JasonGore** [comment]
Chrome + Narrator also does not announcewith [ARIA checkbox reference examples](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/checkbox/checkbox-1.html). I know that Narrator has the most functionality when paired with MS Edge, and significantly diminished functionality with other browsers. MS Edge + Narrator announce the value of the checkboxes at the provided link, so I recommend entering an issue in Narrator's repo. Thanks!
